### DISCUSSION TITLE
Devtools for NgRx

### DISCUSSION THREAD

**Discussion by AmmarSiddiqi2** [reduxjs/redux-devtools#1327] [Q&A]
Why there is no option to enable the trace mood in Redux Devtools for Angular application with NgRx?
Although the docs [https://ngrx.io/guide/store-devtools](url) tell to add trace option to true but in node_modules the StoreDevtoolsConfig does not support trace option.
NgRx version 11.1.1
![image](https://user-images.githubusercontent.com/84870828/211297926-be8c7f72-cedd-4525-90ab-e219a5060c22.png)

**Comment by markerikson** [comment]
@AmmarSiddiqi2 : the `StoreDevtoolsConfig` type appears to be something that's specific to NgRx.  It looks like it is _based_ on the Redux DevTools extension types, but it's not an issue with the Redux DevTools extension itself.  I think you need to file this issue over in the NgRx repo instead.
